simapp/simd/cmd/config.go

```diff
@@ -56,15 +56,17 @@ func initClientConfig() (string, interface{}) {
 		},
 	}
 
-	// The default SDK app template is defined in serverconfig.DefaultConfigTemplate.
-	// We append the custom config template to the default one.
-	// And we set the default config to the custom app template.
-	customClientConfigTemplate := clientconfig.DefaultClientConfigTemplate + strings.TrimSpace(`
+	// Create a customConfig to define specific settings.
+	customConfig := strings.TrimSpace(`
 # This is default the gas adjustment factor used in tx commands.
 # It can be overwritten by the --gas-adjustment flag in each tx command.
 gas-adjustment = {{ .GasConfig.GasAdjustment }}
 `)
 
+	// The CustomizeConfigTemplate function is then employed to seamlessly integrate these custom settings
```

Comment:
If we go that way we will to be consistent with app.toml as well.
Given that chains are already using the app.toml way, maybe we shouldn't change it (?)
If we were to break people, maybe we should remove templates and use the go-toml lib to be able just define a custom struct.

Reply:
Thanks Julien!! You are right, I'm going to rollback this change and add the gRPC as two simple fields